### DISCUSSION TITLE
Oops! An Error Occurred The server returned a "500 Internal Server Error".

### DISCUSSION THREAD

**Discussion by 0MAL** [mosparo/mosparo#251] [Problems & Issues] [answered]
When I enter the username and password for entering mosparo console, I have below error.
I didn't have this error before.
Any ideas?

```
 Oops! An Error Occurred
 The server returned a "500 Internal Server Error".
 Something is broken. Please let us know what you were doing when this error occurred. We will fix it as soon as possible. Sorry for any inconvenience caused.
```

And an error in browser's console is as below.
` Content-Security-Policy: The page's settings blocked the loading of a resource at inline ("style-src").`

**Comment by zepich** [comment]
Hi @0MAL 

Thank you very much for your question.

Can you look into the log file in the directory `<your-mosparo-directory>/var/log/`? There should be a log file with the name `prod-<date>.log`, where you should see a more precise error message.

The Content Security Policy is probably only visible because of this error message, but it should not be the reason for the error in general.

Thank you for your answer.

Kind regards,

zepich

**Comment by 0MAL** [comment]
`[2024-06-18**********] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Lock\Exception\InvalidArgumentException: "The FlockStore directory "/tmp/mydirectory" does not exists and cannot be created." at /mymosparodirectory/vendor/symfony/lock/Store/FlockStore.php line 48 {"exception":"[object] (Symfony\\Component\\Lock\\Exception\\InvalidArgumentException(code: 0): The FlockStore directory \"/tmp/mydirectory\" does not exists and cannot be created. at /mymosparodirectory/vendor/symfony/lock/Store/FlockStore.php:48)"} []`

**Reply by zepich** [reply]
Hi @0MAL 

Thank you very much!

Have you configured the PHP option `sys_temp_dir` in your `php.ini` configuration file?

If yes, does the directory you defined exist and is writable by the PHP process (web server or PHP-FPM, depending on your setup)?

Kind regards,

zepich

**Comment by 0MAL** [comment, marked as answer]
Resolved now. Settings of my server were reset, perhaps by the owner.
Thanks.

**Reply by zepich** [reply]
No problem! :)
